modules/RoyalCommands/src/main/resources/config.yml

```diff
@@ -689,3 +689,25 @@ items:
       # The lore to apply to spawned items. This allows for easy identification without /spawninfo.
       lore:
       - "&9Spawned"
+
+###
+# Death Message Options #
+###
+death:
+  # "use_custom_death" - Should custom death messages be used?
+  use_custom_death: true
```

Comment:
Debating whether we want to stick with `death` as the setting category. The rest are a mix, some plural (`commands`), others encompassing larger functionality (`pluginmanager`). If we think we might implement some kind of death-management behaviours later (eg. instant respawn) it might be better to either rename this to something like `death_alerts`, or keep `death` as the parent and use a subcategory `messages` with `enabled: true/false` rather than `use_custom_death` to be consistent with other areas.

Reply:
I changed it to `deathmessages`, this follows the similar pattern as existing options `messages`, `spawns`, `kits`